src/Components/ProductCard.js

```diff
@@ -0,0 +1,48 @@
+import { useData } from "../data-context";
+import "./styles.css";
+
+export function ProductCard({product}) {
+    let {
+        id,
+        name,
+        image,
+        price,
+        productName,
+        inStock,
+        level,
+        fastDelivery,
+        quantity
+      } = product;
+    const { dispatch } = useData();
+    return(
+
+        <div
+            key={id}
```

Comment:
check out this link: [https://reactjs.org/docs/lists-and-keys.html#extracting-components-with-keys](https://reactjs.org/docs/lists-and-keys.html#extracting-components-with-keys)

It shows correct usage of keys in our components, the `key` shouldn't be here

Reply:
Thanks for sharing the article, it does make sense to pass key directly while calling the component